scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -206,7 +251,7 @@ async def execute_notebook(path: Path, config: Config) -> bool:
     Wrapper function for `_execute_notebook` to print status and write result
     """
     if config.should_patch(path):
-        print(f"▶️ Executing {path} (with least_busy patched to return fake backend)")
+        print(f"▶️ Executing {path} (with least_busy patched to {config.args.backend})")
```

Comment:
I noticed in CI it says we're patching with `fake_athens` even though we're patching with the Qiskit generic backend. I think easiest is to just remove the backend name. We can always check the command to see how we patched it.

```suggestion
        print(f"▶️ Executing {path} (with least_busy patched)")
```